gobblin-utility/src/main/java/gobblin/util/ProxiedFileSystemWrapper.java

```diff
@@ -120,7 +120,7 @@ public Void run() throws IOException {
    * @return Token for proxyUserName if it exists.
    * @throws IOException
    */
-  private Optional<Token> getTokenFromSeqFile(String authPath, String proxyUserName) throws IOException {
+  public static Optional<Token> getTokenFromSeqFile(String authPath, String proxyUserName) throws IOException {
```

Comment:
This class has been deprecated. You can use `ProxiedFileSystemUtils.getTokenFromSeqFile` instead.